mercury/static/mercury/js/simulator.js

```diff
@@ -83,24 +83,26 @@ $(function () {
             current_fuel_level += 90;
         }
         else{
-            current_fuel_level -= parseFloat((Math.random() * 10).toFixed(2));
+            //current_fuel_level -= parseFloat((Math.random() * 10).toFixed(2));
+            current_fuel_level -= getRandomNumber(0,5);
```

Comment:
```suggestion
            current_fuel_level -= getRandomNumber(0,50);
```

Reply:
I think we should be able to have a larger number of fuel than 5

Reply:
fuel will be larger numbers only, the above line 86 is for decreasing the fuel level. I think it's better to decrease the fuel level slowly by a random number between 0 and 5. Also the check I provided is if the fuel level is less than 10, it will be incremented with 90. if we use random number between 0 and 50 the fuel level will go in negative numbers.

Reply:
Oh, I understand. I missed the `-` in the `-=` operator. Good catch.